charts/celestia-local/templates/_helpers.tpl

```diff
@@ -0,0 +1,102 @@
+{{/*
+Namepsace to deploy elements into.
+*/}}
+{{- define "celestia.namespace" -}}
+{{- default .Release.Namespace .Values.global.namespaceOverride | trunc 63 | trimSuffix "-" -}}
+{{- end }}
+
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "celestia.appName" -}}
+{{- default (include "celestia.name" .) | trunc 63 | trimSuffix "-" }}-astria-dev-cluster
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "celestia.labels" -}}
+{{ include "celestia.selectorLabels" . }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "celestia.selectorLabels" -}}
+app: {{ include "celestia.appName" . }}
+{{- end }}
+
+{{/*
+The log level represented as a number
+*/}}
+{{- define "celestia.logLevelNum" -}}
+{{- if eq .Values.config.logLevel "error" }}
+1
+{{- else if eq .Values.config.logLevel "warn" }}
+2
+{{- else if eq .Values.config.logLevel "info" }}
+3
+{{- else if eq .Values.config.logLevel "debug" }}
+4
+{{- else if eq .Values.config.logLevel "trace" }}
+5
+{{- end }}
+{{- end }}
+
+{{/*
+Full image paths for Astria built images
+*/}}
+{{- define "celestia.image" -}}
```

Comment:
Should Astria built images be here?

Reply:
Good catch! Thanks! Actually looks like none of this image stuff is used anywhere (not even in the celestia-node chart). I must have copied from another chart + did search/replace and forgot to remove references that aren't needed.